src/style.css

```
@@ -0,0 +1,96 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("./fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-weight: 600;
+}
+
+html {
+  font-family: "Avenir-Book", Helvetica, sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  padding: 0;
+  margin-top: 20px;
+}
+
+.searchbar {
+  width: 100%;
+  font-family: inherit;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.big_search {
+  height: 70px;
+  margin-bottom: 20px;
+  padding-left: 62px;
+  line-height: 22px;
+  font-size: 16px;
```

Зроби відступи між різними групами CSS правил, тоді читабільність твого коду виросте
```suggestion
  height: 70px;
  margin-bottom: 20px;
  padding-left: 62px;
  
  line-height: 22px;
  font-size: 16px;
```